#!/usr/bin/env python
PACKAGE = "wm_object_segmentation"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("lim_size_gripper", double_t, 0, "Limite object size that we can grip", 0.2, 0, 0.5)
gen.add("lim_max_size_object", double_t, 0, "Limite object size to publish the object", 0.4, 0, 1)
gen.add("lim_distance_object", double_t, 0, "Limite distance to publish an object", 2, 0, 5)

exit(gen.generate(PACKAGE, "wm_object_segmentation", "wm_object_segmentation"))
